Zend/tests/readonly_props/variation.phpt

```diff
@@ -0,0 +1,118 @@
+--TEST--
+Readonly variations
+--FILE--
```

Comment:
Add a string concatenation variation maybe? And maybe ``++``, ``--`` as IIRC those are different opcodes

Reply:
`+=` and `.=` both result in `ASSIGN_OP`. `++` might be worth checking. I'll adjust the test.